lua/kubectl/views/api-resources/init.lua

```diff
@@ -0,0 +1,40 @@
+local ResourceBuilder = require("kubectl.resourcebuilder")
+local definition = require("kubectl.views.api-resources.definition")
+local state = require("kubectl.state")
+local tables = require("kubectl.utils.tables")
+
+local M = {}
+
+function M.View(cancellationToken)
+  local self = state.instance
+  if not self or not self.resource or self.resource ~= definition.resource then
+    self = ResourceBuilder:new(definition.resource)
+  end
+
+  self:display(definition.ft, definition.resource, cancellationToken)
+
+  state.instance = self
+  self.data = cached_resources and cached_resources.values or {}
```

Comment:
cached_resources doesn't seem to be defined here ☝️

Reply:
oops. fixed